js/index.js

```diff
@@ -192,4 +191,50 @@ document.addEventListener('DOMContentLoaded', () => {
     event.preventDefault();
     showError(email.value);
   });
+
+  // Local-Storage
+  // declaring new variables
+  // const storage = [{
+  //   fname: '',
+  //   lname: '',
+  //   mail: '',
+  //   msg: '',
+  // }];
```

Comment:
- [ ] Kindly remove this commented code from the file. That way, your code is clean. 👍